lib/kafka_base_producer.js

```diff
@@ -228,9 +234,10 @@ KafkaBaseProducer.prototype.produce = function produce(topic, message, timeStamp
 
     if (self.closing) {
         if (callback) {
-            callback(new Error('cannot produce to closed KafkaRestProducer'));
-        } // TODO: else { self.logger.error() }
-        return;
+            return callback(new Error('cannot produce to closed KafkaRestProducer'));
+        }
+        // TODO: else { self.logger.error() }
+        return null;
```

Comment:
nit: I think it's more conventional to just `return` (undefined), at least in the uber's codebases I've seen. or is there actually a lint rule around this?

Reply:
Yes this is one lint rule that is complaining. https://eslint.org/docs/rules/consistent-return
Maybe I should change to "return void callback()" and then we are fine.